### PR TITLE
chore(storybook): Add 2 stories for Vuelidate

### PR DESCRIPTION
**Pull request by followyouhome**
### What was a problem?

In https://github.com/Rebilly/rebilly-recomponents/pull/145/ proposed to create separate prop for tag validator in RSelect component. I'd like to propose use only Vuelidate outside of select component.

### How this PR fixes the problem?

Added two stories how to use Vuelidate with Recomponents

### Check lists

- [ ] Readme file updated with actual description and example
- [ ] Added all ARIA attributes required for component
- [ ] Added tests for both browser and SSR render and for all components properties
- [x] Added story with all knobs and actions

### CODE REVIEW

File: packages/recomponents/src/components/r-select/r-select.story.js [hunk 1]
```diff
@@ -516,4 +518,71 @@ storiesOf('Components.Select', module)
...
+                    value: 'hamster',
+                }, {
+                    label: 'Chair',
+                    value: 'chair',
+                }],
+            };
+        },
+        props: {
+            multiple: {
+                default: boolean('Multiple', true),
```
Comment (qqcky) [inline packages/recomponents/src/components/r-select/r-select.story.js]: wasn't used in story

File: packages/recomponents/src/components/r-select/r-select.story.js [hunk 2]
```diff
@@ -516,4 +518,71 @@ storiesOf('Components.Select', module)
...
+        computed: {
+            label() {
+                return `Select only pets: ${this.$v.$invalid ? 'invalid' : 'valid'}`;
+            },
+        },
+        mixins: [validationMixin],
+        validations: {
+            value: {
+                required,
+                pet: helpers.regex('alpha', /(cat|hamster|doge)/),
```
Comment (qqcky) [inline packages/recomponents/src/components/r-select/r-select.story.js]: will it work for multiple? shouldn't we use `$each`?

Comment (followyouhome) [reply packages/recomponents/src/components/r-select/r-select.story.js]: Nope, but this was not the goal - for multiple we can create one more story or update this one.

File: packages/recomponents/src/components/r-select/r-select.story.js (hunk 1 above)
Comment (followyouhome) [reply packages/recomponents/src/components/r-select/r-select.story.js]: updated

**Pull request merged**